audio/tauonmb/tauonmb.SlackBuild

```diff
@@ -76,7 +76,9 @@ mkdir -p lib
 mv libphazor.so lib/libphazor.so
 
 # Compile translator
-python3 compile-translations.py install --root=$PKG
+python3 compile-translations.py
+( cd locale
+  rm -f */*/tauon.po )
```

Comment:
Instead of spawning a subshell, do this:
rm -f locale/*/*/tauon.po

Reply:
Good idea, right away